From 29b5bff6a26180c2403441a81a0d0633c262c44c Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Tue, 26 Jun 2018 15:52:44 +0200
Subject: [PATCH] Revert "sepolicy: qcom: Allow nfc to read and execute files
 in /vendor on full treble"

This reverts commit ea928814709e8e022e43cde9dc48ca4dce6d881a.

Change-Id: I61e282f93b0f4427a58bd134af2f4b0e6cb7cb76
---
 qcom/common/nfc.te | 3 ---
 1 file changed, 3 deletions(-)
 delete mode 100644 qcom/common/nfc.te

diff --git a/qcom/common/nfc.te b/qcom/common/nfc.te
deleted file mode 100644
index 1bd8720..0000000
--- a/qcom/common/nfc.te
+++ /dev/null
@@ -1,3 +0,0 @@
-full_treble_only(`
-    allow nfc vendor_file:file rx_file_perms;
-')
-- 
2.17.1

